src/components/modern/Button.js

```diff
@@ -37,7 +37,7 @@ type Size = 'regular' | 'large' | 'compact';
 
 type Props = {|
   title?: string,
-  onPress: () => mixed,
+  onPress?: () => mixed,
```

Comment:
I would keep the prop mandatory, as normally buttons need onPress handler, I cannot think any reasonable scenario when we might want to have a button without press event handler. Wdyt?

Reply:
@mdjastrzebski This is the done for the case where we need regular disabled swipe up button. However if we keep onPress as optional, the code doesn't break.